src/mgs/mgs_parser.rs

```diff
@@ -126,7 +169,52 @@ where
                 param,
                 value,
             }),
+            MgsStat::FsNames(value) => TargetStats::FsNames(TargetStat {
+                kind: TargetVariant::MGT,
+                target,
+                param,
+                value,
+            }),
         })
         .map(Record::Target)
         .message("while parsing mgs params")
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use combine::{many, parser::EasyParser};
+    use insta::assert_debug_snapshot;
+
+    #[test]
+    fn test_params() {
```

Comment:
There should ideally be three tests here:

1. Test output with a single mounted fs
1. Test output with multiple mounted fs
1. Test output with no mounted fs

In addition, the tests should be smaller to just test the new parser in isolation.

That's not to say this larger test needs to be removed as it's useful as well.

Reply:
Sure. I didn't see a test for the mgs so I figured I would write a test that included all of the output. I'll add a couple of tests to cover the cases listed above.

Reply:
This has been updated in the new module.